### ISSUE TITLE
Does it support Aarch64？

### ISSUE THREAD

**Issue by UMAHunter** [yanqi27/core_analyzer#94]
Hi, Is there a plan for architecture Aarch64?

**Comment by yanqi27** [comment]
@UMAHunter 
CPU architecture is a small factor to this tool. It is more related to the platform and the runtime heap memory manager. Which one do you have in your mind, Linux/ptmalloc, or something else?

**Comment by UMAHunter** [comment]
When I use 'decode' command in an aarch64 environment, it reports an error with message "../../gdb/dwarf2-frame.c:1324: internal-error: Unknown register rule.". Most other commands do not report error. So is this problem involes with the ARM CPU or the OS?

**Comment by yanqi27** [comment]
Yes, you are right. `decode` function is intended to add object context to the disassemble function. It is tied to CPU architecture.
